trin-core/src/portalnet/protocol.rs

```diff
@@ -136,6 +136,7 @@ impl PortalnetEvents {
         }
 
         let request = match Message::from_bytes(talk_request.body()) {
+            // we would need to add an Ok(Message::Response(Response::Accept(r))) in order to accept Accept responses
```

Comment:
I think there's a whole conversation that needs to happen here. We don't really do anything with responses at all anywhere. So this comment can be removed and created as an issue to: design an API for decoding responses. The suggestion in this comment can be added to the body. I think there are some significant alternatives to consider.

Reply:
I originally had this comment in for thinking about how nodes would then respond to the `ACCEPT` message, but since we are dropping `STORE`, I don't need it. But I guess it is still an option to handle responses if there is no where else that does it.

> We don't really do anything with responses at all anywhere.

Yeah I was racking my head thinking about where the code to handle response messages like `FindContent` and `FindNodes` are. Has this not been implemented in Trin yet or am I blatantly missing that section?

Reply:
> Yeah I was racking my head thinking about where the code to handle response messages like `FindContent` and `FindNodes` are. Has this not been implemented in Trin yet or am I blatantly missing that section?

Nope, not really implemented yet. Discv5 returns the talkreq response as a slice of bytes, and right now we're not doing anything beyond returning that (we're not parsing the result at all). For example:

https://github.com/ethereum/trin/blob/5c5a1b3015451e402781fd349a7f3e296cf2639b/trin-core/src/portalnet/protocol.rs#L269-L282